src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindowSettings.cs

```diff
@@ -66,6 +71,26 @@ public NativeWindowSettings()
         /// <inheritdoc />
         public string ClipboardString { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value representing the current graphics API.
+        /// </summary>
+        public ContextAPI API { get; set; }
```

Comment:
/// When changing this to values other than OpenGL, the Version will need to be set appropriately as well.